Moodis/Moodis/Feature/Menu/MenuForm.cs

```diff
@@ -57,21 +61,38 @@ public async void UpdateLabels()
                MessageBox.Show(WarningFaceDetection);
             }
         }
-
         private void BtnCalendar_Click(object sender, EventArgs e)
         {
             Hide();
             var calendarForm = new CalendarForm(new CalendarViewModel(), this);
             calendarForm.Show();
-        }
+            player.StopMusic();
 
+        }
         private void MenuFormClose(object sender, FormClosingEventArgs e)
         {
             if (e.CloseReason == CloseReason.UserClosing)
             {
                 e.Cancel = true;
                 Hide();
+                player.StopMusic();
+            }
+        }
+        private void ButtonMusicController_Click(object sender, EventArgs e)
+        {
+            double highestEmotion = 0;
+            int counter = 0;
+            int index = -1;
+            foreach (var label in menuViewModel.currentImage.emotions)
```

Comment:
rename label to something more meaningfull, i though you began to take info from lbl.Text.